content/components/relative-time.mdx

```diff
@@ -1,16 +1,24 @@
 ---
-title: Relative time
+title: Dates & Times
 status: Experimental
 description: A relative time displays the time difference between the current time and the specified date-time.
 ---
 
 import {Box} from '@primer/react'
 
-## Content
+## Overview
+
+All dates and times displayed should use the Relaive Time component, which is able to format and localise dates into the users given locale, as well display times relative to the current date, while also updating in real time.
+
+Dates and times almost always appear relative to the current date, using short-hand formats to help users quickly determine how old something is or how soon something will be. For this we use Relative Time which displays dates rounded to a nearest unit, for example "a minute ago" or "20 days ago". For dates that are more than 1 month from the current date it can be more useful to display the date-time, as these items are less "current" and more "historical".
```

Comment:
This is great ❤️!  some nit-pick suggestions: 

```suggestion
Dates and times almost always appear relative to the current date, using short-hand formats to help users quickly determine how old something is or how soon something will be. For this, we use Relative Time which displays dates rounded to the nearest unit, for example, "a minute ago" or "20 days ago". For dates that are more than 1 month from the current date, it can be more useful to display the date-time, as these items are less "current" and more "historical".
```